sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3473,8 +3502,8 @@ object functions {
    *   df.selectExpr("filter(col, (x, i) -> i % 2 == 0)")
    * }}}
    *
-   * @param column: the input array column
-   * @param f: (col, index) => predicate, the Boolean predicate to filter the input column
+   * @param column the input array column
+   * @param f: (col, index) => predicate the Boolean predicate to filter the input column
```

Comment:
we should also remove `:` here too.

Reply:
How about `@param f a boolean predicate to filter the input column. Its signature is (col, index) => predicate`

Reply:
It's a little wordy from my view..

Reply:
then let's just remove `:`. Javadoc doesn't work well with it.

Reply:
Yea, I was about to point out. Just removing `:` should be good enough.